### PR TITLE
COMMERCE-1961 Reset form validation if they're changing to an existing address

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Review by ethanbustad — APPROVED** [review]

File: commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp [hunk 1]
```diff
@@ -188,6 +188,8 @@ long commerceRegionId = BeanParamUtil.getLong(currentCommerceAddress, request, "
 				}
 				else {
 					<portlet:namespace />updateAddressFields(commerceAddress.get('selectedIndex'));
+
+					Liferay.Form.get("<portlet:namespace />fm").formValidator.validate();
```
Comment (ethanbustad) [inline commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp]: Actually can you change these to single quotes? Looks like every other JS string in this file is single-quoted, so it'd be great to keep it consistent.

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by ethanbustad** [comment]
ugh, baseline issues in portal. gonna just have to wait for them to be resolved I think

**Comment by alecsloan** [comment]
ci:test

**Pull request merged**